sites/partners/src/pages/index.tsx

```diff
@@ -420,6 +466,83 @@ export default function ListingsList() {
                   </div>
                 </Grid.Cell>
               </Grid.Row>
+              {isNonRegulatedEnabled && (
```

Comment:
@emilyjablonski here is the version using two "Field" and below is the commented out version with the "FieldGroup".  In that version the description is within an expandable section

Reply:
Looks good! Before is `isNonRegulatedEnabled` line, could we just wrap this entire section in a div with `aria-live="polite"` so that the section appearing is announced?